### DISCUSSION TITLE
Call-sSNV Algorithms Comparison

### DISCUSSION THREAD

**Discussion by maotian06** [uclahs-cds/pipeline-call-sSNV#123] [General]
Here is the comparison of SNV in different algorithms using CPCG0196 and one of my SACC WXS samples, I will update the A-full results later:

CPCG0196 WGS:
| count | category                            | algorithm.count | compare.group |
|-------|-------------------------------------|-----------------|---------------|
|  5529 | Strelka2                            |               1 |             1 |
| 30407 | Mutect2                             |               1 |             2 |
|   977 | MuSE                                |               1 |             3 |
|  1177 | SomaticSniper                       |               1 |             4 |
|  2063 | Mutect2-Strelka2                    |               2 |            12 |
|   132 | MuSE-Strelka2                       |               2 |            13 |
|    19 | SomaticSniper-Strelka2              |               2 |            14 |
|   172 | MuSE-Mutect2                        |               2 |            23 |
|     6 | Mutect2-SomaticSniper               |               2 |            24 |
|     4 | MuSE-SomaticSniper                  |               2 |            34 |
|  1021 | MuSE-Mutect2-Strelka2               |               3 |           123 |
|    21 | Mutect2-SomaticSniper-Strelka2      |               3 |           124 |
|    21 | MuSE-SomaticSniper-Strelka2         |               3 |           134 |
|    10 | MuSE-Mutect2-SomaticSniper          |               3 |           234 |
|  1047 | MuSE-Mutect2-SomaticSniper-Strelka2 |               4 |          1234 |

<img width="500" alt="image" src="https://user-images.githubusercontent.com/47042983/193642896-478654f4-59b1-4d85-bfce-b2237ea34578.png">

SACC WXS:
| count | category                            | algorithm.count | compare.group |
---|---|---|---
|  2217 | SomaticSniper                       |               1 |             1 |
|   607 | MuSE                                |               1 |             2 |
|   557 | Mutect2                             |               1 |             3 |
|  3117 | Strelka2                            |               1 |             4 |
|    37 | MuSE-SomaticSniper                  |               2 |            12 |
|     7 | Mutect2-SomaticSniper               |               2 |            13 |
|   697 | SomaticSniper-Strelka2              |               2 |            14 |
|   108 | MuSE-Mutect2                        |               2 |            23 |
|  1086 | MuSE-Strelka2                       |               2 |            24 |
|   588 | Mutect2-Strelka2                    |               2 |            34 |
|     9 | MuSE-Mutect2-SomaticSniper          |               3 |           123 |
|  1957 | MuSE-SomaticSniper-Strelka2         |               3 |           124 |
|   207 | Mutect2-SomaticSniper-Strelka2      |               3 |           134 |
|  1020 | MuSE-Mutect2-Strelka2               |               3 |           234 |
|  2863 | MuSE-Mutect2-SomaticSniper-Strelka2 |               4 |          1234 |

<img width="500" alt="image" src="https://user-images.githubusercontent.com/47042983/193395894-b4067bc2-0d6e-4725-8971-be744150c188.png">

**Comment by sorelfitzgibbon** [comment]
Just want to note that I came across the following discussion where the SomaticSniper author says he doesn't recommend using Sniper anymore other than for legacy reasons: "Sniper was a first-gen caller, and more modern callers generally have much improved sensitivity and specificity."

https://github.com/genome/somatic-sniper/issues/10

**Reply by pboutros** [reply]
Yup, I remember seeing that during PCAWG -- but empirically SS outperforms most other tools in that it has very high specificity.  I think those results are buried somewhere in the PCAWG marker paper & its supplementary!
